src/clientlayers/ConnectionRequest.jl

```diff
@@ -126,8 +127,10 @@ function connectionlayer(handler)
                     break
                 end
             end
-            if logerrors && !(e isa StatusError || e isa TimeoutError)
-                @error "HTTP.ConnectionRequest" exception=(e, catch_backtrace()) method=req.method url=req.url context=req.context
+            root_err = e isa CapturedException ? e.ex : e
+            if logerrors && !(root_err isa StatusError || root_err isa Exceptions.TimeoutError || root_err isa Base.IOError)
```

Comment:
I'm not really sure how to tell if this list of exceptions is right... Would it make sense to only log the error if the exception is not a `CapturedException` (presumably, if it is a CapturedException, we already logged it before we wrapped it)?

Reply:
Yeah, I like that better